files/aliases

```diff
@@ -0,0 +1,11 @@
+# Control ls
+alias ls='ls --color=auto'
+alias ll='ls --color=auto -l'
+alias l='ls --color=auto -lA'
```

Comment:
Is this docker image intended to be accessed by humans ?

Reply:
It could be. I mean that it's pretty useful for debugging purpose to have such an image. (Especially with py36 & java installed) But it's only my opinion, therefore I could remove this part if it's deemed useless.

Reply:
nope nope, just wanted to be sure. Keep, it, it will be useful indeed.